homeassistant/components/starlink/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""The Starlink integration."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_URL, Platform
+from homeassistant.core import HomeAssistant
+
+from .const import CONFIG_POLL_RATE, DOMAIN
+from .coordinator import StarlinkUpdateCoordinator
+
+PLATFORMS: list[Platform] = [Platform.SENSOR, Platform.BINARY_SENSOR, Platform.BUTTON]
```

Comment:
Please limit the initial PR to a single platform. See also:

<https://developers.home-assistant.io/docs/creating_component_code_review/#5-make-your-pull-request-as-small-as-possible>